image_obj_parse.go

```diff
@@ -280,9 +282,9 @@ func parsePng(f *bytes.Reader, info *imgInfo, imgConfig image.Config) error {
 			if err != nil {
 				return err
 			}
-		} else if string(typ) == "IEND" {
+		case "IEND":
 			break
```

Comment:
in old code this "break" stop for-loop (at line 226), but in the new code this "break" stop switch-case and not stop for-loop anymore.

Reply:
But i can see outer break in golang source:

Reply:
Thank you. :blush: for the pull request.
"outer-break" in my opinion it makes code harder to read ( not much but still harder ) so I think don't use "outer-break" should be better.